modules/grpc/bigquery/bigquery-dest.cpp

```diff
@@ -184,6 +234,68 @@ DestinationDriver::construct_schema_prototype()
   this->schema_prototype = msg_factory.GetPrototype(this->schema_descriptor);
 }
 
+bool
+DestinationDriver::load_protobuf_schema()
+{
+  this->protobuf_schema.src_tree = std::make_unique<google::protobuf::compiler::DiskSourceTree>();
+  this->protobuf_schema.src_tree->MapPath(this->protobuf_schema.proto_path, this->protobuf_schema.proto_path);
```

Comment:
[optional]

If we want to support including in the proto files, we can do that similar to this (now I can include from `/home/alltilla/repos/syslog-ng/`):
```cpp
  this->protobuf_schema.src_tree->MapPath("", "/home/alltilla/repos/syslog-ng");
```

Reply:
Actually `Import()` works with the absolute path, too, even with my change, I have tried it.

But nevermind, we don't need including fortunately.